include/edge.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Renjie Wu on 2019-02-24.
+//
+
+#ifndef PIPELINE_GRAPH_EDGE_H
+#define PIPELINE_GRAPH_EDGE_H
+
+#include <utility>
+#include <memory>
+#include "node.h"
```

Comment:
Have a space between these <> and "" kinda includes. Then run `clang-format` to format the code. I can show you this.